lib/v2/tmdb/episodes.js

```diff
@@ -0,0 +1,27 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const apiKey = require('./api-key');
+const { handleDescription } = require('./utils');
+
+module.exports = async (ctx) => {
+    const { id, seasonNumber, lang } = ctx.params;
+    const searchParams = {
+        language: lang,
+        api_key: apiKey(),
+    };
+    const { data: tvShowDetails } = await got(`https://api.themoviedb.org/3/tv/${id}`, { searchParams });
+    const { data: seasonDetails } = await got(`https://api.themoviedb.org/3/tv/${id}/season/${seasonNumber}`, { searchParams });
+
+    ctx.state.data = {
+        title: `${tvShowDetails.name} ${seasonDetails.name} — TMDB`,
+        description: seasonDetails.overview.trim(),
+        image: `https://image.tmdb.org/t/p/original${seasonDetails.poster_path}`,
+        link: `https://www.themoviedb.org/tv/${tvShowDetails.id}/season/${seasonDetails.season_number}`,
+        item: seasonDetails.episodes.reverse().map((item) => ({
```

Comment:
No need to `reverse()` since `item` will be sorted in reverse chronological order according to their `pubDate`.